modules/cpmso-yaml-provider-spec-vsphere.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control-plane-machine-management.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-provider-spec-vsphere_{context}"]
+= Sample vSphere provider specification
```

Comment:
This section is included as a stub to show my planned framework, and will be commented out for merge, then completed in a later PR.

Placeholder content is AWS.